### DISCUSSION TITLE
Welcome to dictionary Discussions!

### DISCUSSION THREAD

**Discussion by petermr** [petermr/dictionary#5] [General]
## Welcome!
Github has just started Discussions and we are trying it out.

This is the for those in the openVirus community (https://github.com/petermr/openvirus) community  who are interested in Dictionaries

**Comment by petermr** [comment]
We have also use the Github wiki (https://github.com/petermr/dictionary/wiki) for discussing how to develop code and resources. It's possible that we shall transfer some of this to `https://github.com/petermr/dictionary/discussions`. My guess is that the discussions are fairly linear (i..e don't have separate topics, but let's see)

**Reply by AmbrineH** [reply]
Yes
